addons/stock_account/report/report_stock_forecasted.py

```diff
@@ -11,10 +11,22 @@ class ReplenishmentReport(models.AbstractModel):
     def _compute_draft_quantity_count(self, product_template_ids, product_variant_ids, wh_location_ids):
         """ Overrides to computes the valuations of the stock. """
         res = super()._compute_draft_quantity_count(product_template_ids, product_variant_ids, wh_location_ids)
-        domain = self._product_domain(product_template_ids, product_variant_ids)
```

Comment:
Why don't you use `_product_domain()` again ?

Reply:
Because here, the domain is for `product.product`, and the domain returned by `_product_domain` is used to search `stock.move` (so, it's on `product_tmpl_id` in both case for product template, but for variant it's for the `id` field or for the `product_id` field).